packages/front/src/AdminJobs.css

```
@@ -49,3 +49,47 @@
+  align-items: baseline;
+  margin-bottom: 4px;
+}
+
+.admin-jobs-run-result {
+  margin: 0;
+  padding: 8px 10px;
+  background: rgba(127, 127, 127, 0.12);
+  border-radius: 4px;
+  font-size: 0.85em;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.11.1)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)